Homarus/cfg/config.example.yaml

```diff
@@ -0,0 +1,25 @@
+---
+homarus:
```

Comment:
This is missing stuff from the mimetypes and mime_to_format entries from the `config.yaml` you deleted.  You should be able to make that mime_to_format array a map with mimetypes as keys and formats as values to avoid the string manip.